docs/src/main/paradox/scala/http/routing-dsl/directives/parameter-directives/index.md

```diff
@@ -5,27 +5,27 @@
 @@@ index
 
 * [parameter](parameter.md)
+* [parameters](parameters.md)
 * [parameterMap](parameterMap.md)
 * [parameterMultiMap](parameterMultiMap.md)
-* [parameters](parameters.md)
 * [parameterSeq](parameterSeq.md)
 
 @@@
 
 <a id="which-parameter-directive"></a>
 ## When to use which parameter directive?
 
-Usually, you want to use the high-level @ref[parameters](parameters.md) directive. When you need
+Usually, you want to use the high-level @scala[@ref[parameters](parameters.md)]@java[@ref[parameter](parameter.md)] directive. When you need
```

Comment:
We could use `parameter` also for scala as it is just an alias for `parameters`.

Reply:
Or the other way round, maybe link both to parameters, as after this PR, it would contain quite some more info and examples.

Reply:
Makes sense. I added a commit applying this suggestion.

Reply:
Cool :+1: